docs/guides/menus.md

```diff
@@ -43,6 +43,17 @@ Or pass nothing. This is good if you have only one menu. In that case Timber wil
 $menu = new TimberMenu();
 ```
 
+## Options
+
+Optionally, you can send additional options to `Timber\Menu`. Current only `depth` is supported
+
+```php
+$args = array(
+	'depth' => 2,
+);
+$menu = new TimberMenu( 'primary', $args);
+```
+
```

Comment:
Would you mind to add some more information about what the `depth` parameter is doing exactly? I think for many users it might not be clear that `depth` means the maximum depth for a menu.

Reply:
Sure. I'll try to be more verbose... Suggestions on direction there? This really does follow the same flow as the WP menu, so those who are using Timber, I would hope already know the typical way for handling WP's menu... but I am willing to try to explain it a little more clear than it is now.

Reply:
It’s always hard to assume how much users know.

I think it doesn’t need a lot more information. If the options replicate the functionality of the options that can be passed to [`wp_nav_menu()`](https://developer.wordpress.org/reference/functions/wp_nav_menu/), I think it would be enough to mention that and provide the link to the WordPress Code Reference, so that people can read about the options there.